### DISCUSSION TITLE
Do we have to add auth for the browser extension task ?

### DISCUSSION THREAD

**Discussion by Skyhero-alt** [open-sauced-craftwork/browser-extension#45] [Q&A] [answered]
Do we have to add auth for the browser extension task ? If yes then where can I get the Auth token ?

**Comment by Rishav1707** [comment, marked as answer]
> Do we have to add auth for the browser extension task ? If yes then where can I get the Auth token ?

- Visit https://hot.opensauced.pizza/ and sign in with the GitHub account.
- After signing in, click on your profile and choose the third option which is `print auth token`.
- After clicking that option check the dev tools console.

**Reply by Skyhero-alt** [reply]
Thanks a lot
